chore: export components declaration (#2189)
TuSimple/naive-ui: demo/pages/docs/usage-sfc/enUS/index.md

  <n-button>naive-ui</n-button>
</template>
```

### Volar support

> If you already use Volar, you specify global component types by configure `compilerOptions.types` in `tsconfig.json`.

Reviewer: ```suggestion
If you are using Volar, you can specify global component types by configure `compilerOptions.types` in `tsconfig.json`.
```
